src/gurt/fault_inject.c

```diff
@@ -230,7 +230,7 @@ int
 d_fault_attr_err_code(uint32_t fault_id)
 {
 	struct d_fault_attr_t	*fault_attr;
-	uint32_t		 err_code;
+	int32_t			 err_code;
```

Comment:
Also don't need the case in return;

Reply:
The cast?  I did wonder about that.  I'll remove.

Reply:
Line 245: return (int) err_code;